styles.css

```diff
@@ -69,3 +110,135 @@
 .contact-info-button {
   margin-right: 30px;
 }
+
+/* Recent works */
+.recent-works {
+  margin-bottom: 40px;
+  display: flex;
```

Comment:
- [ ] Well done on using flexbox here but please use display:grid for the main container of the projects as it was part of the [project requiremtens](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md). Keep rocking 💪